rebar.config

```diff
@@ -34,23 +29,115 @@
   {seestar, ".*", {git, "git://github.com/arcusfelis/seestar.git", "e9700e1e7e8226173ad76d22be9d3a4e0ef77310"}},
 
   {cache_tab, ".*", {git, "git://github.com/processone/cache_tab", {tag, "1.0.4"}}},
-  {stringprep, ".*", {git, "git://github.com/processone/stringprep.git", {tag, "1.0.6"}}},
+  {stringprep, ".*", {git, "git://github.com/erszcz/stringprep.git", {branch, "list-deps"}}},
 
   {proper, ".*", {git, "git://github.com/manopapad/proper.git", {tag, "v1.2"}}},
   {meck, ".*", {git, "git://github.com/eproxus/meck.git", {tag, "0.8.3"}}},
 
   {pa, ".*", {git, "git://github.com/erszcz/pa.git", "e53c540"}},
   {usec, ".*", {git, "git://github.com/esl/usec.git", {branch, "master"}}},
-  {ecoveralls, ".*", {git, "git://github.com/nifoc/ecoveralls.git", "0e52c47"}},
   {edown, ".*", {git, "git://github.com/uwiger/edown.git", {tag, "0.8"}}},
-  {mustache, ".*", {git, "git://github.com/mojombo/mustache.erl.git", "031c7aa"}},
-  {recon, ".*", {git, "git://github.com/ferd/recon.git", "2.3.2"}}
+  {bbmustache, ".*", {git, "git://github.com/soranoba/bbmustache.git", {tag, "v1.4.0"}}},
+  {recon, ".*", {git, "git://github.com/ferd/recon.git", "2.3.2"}},
+
+  %% TODO: Remove this override of an exometer_core dependency
+  %%       once it bundles a new enough (i.e. with verify_directories=false) version.
+  {setup, {git, "git://github.com/uwiger/setup.git", {tag,"1.6"}}}
+ ]}.
+
+{relx, [{release, { mongooseim, {cmd, "cat VERSION | tr -d '\r\n'"} },
+         []}, %%Apps list is dynamicaly set by rebar.config.script
+
+        {dev_mode, true},
+        %% TODO: extra copies waste space, but mongooseim script requires the files in a certain place
+        {include_erts, true},
+
+        {generate_start_script, false},
+        {extended_start_script, false},
+
+        {overlay, [
+                   {mkdir, "priv/ssl"},
+                   {copy, "fake_cert.pem", "priv/ssl/fake_cert.pem"},
+                   {copy, "fake_key.pem", "priv/ssl/fake_key.pem"},
+                   {copy, "fake_server.pem", "priv/ssl/fake_server.pem"},
+                   {copy, "fake_dh_server.pem", "priv/ssl/fake_dh_server.pem"},
+
+                   {copy, "rel/files/erl", "erts-\{\{erts_vsn\}\}/bin/erl"},
+                   {copy, "rel/files/nodetool", "erts-\{\{erts_vsn\}\}/bin/nodetool"},
+
+                   {template, "rel/files/mongooseim", "bin/mongooseim"},
+                   {template, "rel/files/mongooseimctl", "bin/mongooseimctl"},
+                   {template, "rel/files/app.config", "etc/app.config"},
+                   {template, "rel/files/vm.args", "etc/vm.args"}
+                  ]}
+
+       ]}.
+
+{profiles, [ {default, [{relx, [ {overlay_vars, "rel/vars.config"},
```

Comment:
Right, default doesn't seam to work for release. For "dev" we have more profiles: `mim1`, `mim2`, `mim3` and `fed1`.